docs/source/user_guide/index.rst

```diff
@@ -53,10 +53,10 @@ Lets generate a Gaussian pulse at focus, propagate it backwards by one Rayeligh
    laser = Laser(dimensions,lo,hi,num_points,laser_profile)
 
 ..  code-block:: python
-   :caption: Propagate the laser pulse backwards by one Rayeligh length.
+   :caption: By default, the laser antenna will emit on the focal plan. (Optional) Make the laser antenna emit outside the focal plan by one Rayleigh length.
 
    z_R            = 3.14159*spot_size**2/wavelength    # The Rayleigh length.
-   laser.propagate(-z_R)
+   laser.propagate(-z_R)                               # The laser antenna emits behind the focal plane
```

Comment:
Likewise, this step actually does a numerical propagation by 1 large time step, and this is not use an antenna per se, so I think this wording could be confusing.